blazor/data/adaptors.md

```diff
@@ -213,6 +213,745 @@ The sample response object should look like below.
+
+You can provide the GraphQL query string by using the `Query` property of the `GraphQLAdaptorOptions`. Also you need to set the `ResolverName` property of `GraphQLAdaptorOptions` to map the response. The GraphQLAdaptor expects response as a JSON object with properties of Result, Count and Aggregates which contains the collection of entities, total number of records and value of aggregates respectively. The GraphQL response should be returned in JSON format like { “data”: { … }} with query name as field.
+
+```cshtml
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Data
+@using Syncfusion.Blazor.Grids
+
+<SfGrid TValue="Order" AllowPaging=true AllowSorting=true AllowFiltering=true>
+    <SfDataManager Url="https://xxxxxx" GraphQLAdaptorOptions=@adaptorOptions Adaptor="Adaptors.GraphQLAdaptor"></SfDataManager>
+    <GridColumns>
+        <GridColumn Field=@nameof(Order.OrderID) HeaderText="Order ID" IsPrimaryKey=true TextAlign="TextAlign.Right" Width="120"><GridColumn>
+        <GridColumn Field=@nameof(Order.CustomerID) HeaderText="Customer Name" Width="150"></GridColumn>
+        <GridColumn Field=@nameof(Order.OrderDate) HeaderText="Order Date" Format="d" TextAlign="TextAlign.Right"
+        Width="130"></GridColumn>
+        <GridColumn Field=@nameof(Order.Freight) HeaderText="Freight" Format="C2" TextAlign="TextAlign.Right" Width="120"></GridColumn>
+    </GridColumns>
+</SfGrid>
+
+@code{
+    private GraphQLAdaptorOptions adaptorOptions { get; set; } = new GraphQLAdaptorOptions
+    {
+        Query = @"
+            query ordersData($dataManager: DataManagerRequestInput!){
+                ordersData(dataManager: $dataManager) {
+                    count, result { OrderID, CustomerID, OrderDate, Freight } , aggregates
+                }
+            }",
+        ResolverName = "OrdersData"
+    };
+
+    public class Order
+    {
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public DateTime? OrderDate { get; set; }
+        public double? Freight { get; set; }
+    }
+}
+```
+
+Configuration in GraphQL server application to set GraphQL query and mutation type and to enable CORS.
+
+Program.cs
+
+```cshtml
+
+var builder = WebApplication.CreateBuilder(args);
+//GraphQL resolver is defined in GraphQLQuery class and mutation methods are defined in GraphQLMutation class
+builder.Services.AddGraphQLServer().AddQueryType<GraphQLQuery>().AddMutationType<GraphQLMutation>();
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowSpecificOrigin", builder =>
+    {
+        builder.WithOrigins("https://xxxxxx")
+        .AllowAnyHeader()
+        .AllowAnyMethod()
+        .AllowCredentials().Build();
+    });
+});
+
+```
+
+Below is the resolver function in the GraphQL server used to bind data.
```

Comment:
need to explain each functionality separately. Here shared entire code it is not understandable